patches/@jsdevtools__ono@7.1.3.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/esm/index.js b/esm/index.js
```

Comment:
This is the same patch for `@jsdevtools/ono` as we built for synapse-react-client (can be seen in `synapse-web-monorepo/patches`)

Other folks have experienced the same issue: https://github.com/JS-DevTools/ono/issues/19